woocommerce/woocommerce
# Fix Home screen grey background

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR fixes the grey background on the home screen. Before this fix, it disappeared when navigating from the product editor.

Before

![Screen Recording on 2024-02-26 at 11-08-05](https://github.com/woocommerce/woocommerce/assets/1314156/3ed46dd8-5cfe-46de-81cc-3cc449e75c06)

After

![Screen Recording on 2024-03-25 at 11-08-48](https://github.com/woocommerce/woocommerce/assets/1314156/d194a3e0-87fd-4c58-88fb-f2cf2a508cfc)

Closes #41308.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Make sure the New product editor is enabled under `/wp-admin/admin.php?page=wc-settings&tab=advanced&section=features`.
2. Go to Product > Add New.
3. Then go to WooCommerce > Home.
4. Verify that the Home screen background color is grey.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
